packages/brick-kit/src/core/CustomForms/constants.ts

```diff
@@ -1 +1 @@
-export const formRender = "form-builder.form-render";
+export const formRender = "form-render.form-render";
```

Comment:
`render` 是动词，渲染器应该用 `renderer`

```suggestion
export const formRenderer = "form-renderer.form-renderer";
```